lidarr/tag_test.go

```diff
@@ -61,7 +61,7 @@ func TestGetTag(t *testing.T) {
 	tests := []*starr.TestMockData{
 		{
 			Name:           "200",
-			ExpectedPath:   path.Join("/", starr.API, lidarr.APIver, "tag/1"),
+			ExpectedPath:   path.Join("/", starr.API, lidarr.APIver, "tag", "1"),
```

Comment:
woah you didn't ave to do all that. My comments on things like this mostly nitpicks.

Reply:
I just did find and replace and it came across many files